### PR TITLE
Add ramen-dr job

### PR DESCRIPTION
**Pull request by rakeshgm**
This is an initial attempt to run ramendr/ramen e2e tests which should trigger for every PR. 
This PR will only run unit tests for now. 
Ramen Repo: https://github.com/RamenDR/ramen.git

### CODE REVIEW

File: ramen-pull-requests/config/definations/ramen-pull-requests.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 30
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/rakeshgm/ramen/
```
Comment (rakeshgm) [inline ramen-pull-requests/config/definations/ramen-pull-requests.yaml:19]: I have added my fork for testing purposes

File: ramen-pull-requests/config/definations/ramen-pull-requests.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - github-pull-request:
+          trigger-phrase: "jenkins test.*|jenkins retest.*"
+          only-trigger-phrase: false
+          github-hooks: true
+          permit-all: false
+          auto-close-on-fail: false
+
+    scm:
+      - git:
+          url: https://github.com/rakeshgm/ramen.git
```
Comment (rakeshgm) [inline ramen-pull-requests/config/definations/ramen-pull-requests.yaml:36]: added my fork for testing purposes

**Comment by dmick** [comment]
Good start.  You'll need to clean up anything you install, too.  A few notes:

* is installing go from a tarball really the right way to install go?  Are there not distro packages?

* if you make the git repo a variable, you can give a default of your workspace for now, and then just change it when you're ready with the job

* have you tested build/build on a clean installed image (container or vm, maybe)?

**Comment by rakeshgm** [comment]
so installation from Tarball is what the official document says. [doc](https://go.dev/doc/installl). 
I will make the remaining changes

**Review by dmick — COMMENTED** [review]
What triggers this build?

**Comment by rakeshgm** [comment]
> What triggers this build?

mostly pull-requests to the repo.

**Review by dmick — CHANGES_REQUESTED** [review]
I pulled the branch locally and tested pushing it to the server.  Noticed a few minor things:

1) name the file .yml like all the others (rather than .yaml)
2) Title the pull request "Add ramen-pull-requests" (which is the name of the actual job)
3) change the commit message to describe what the job does and for what purpose; the fact that it's the initial change is obvious from the git history, so note what ramen is and what role it serves in ceph

**Comment by dmick** [comment]
I pushed the job to the jenkins server by hand.  It failed because it ran on a non-dnf system; you'll need to add labels to select the right sort of build host.  I suspect you want something like either 'centos8' or 'centos9'.  Also, I doubt it needs to be "huge".  Does/will this need testing on both x86_64 and arm64?

Here's the build log: https://jenkins.ceph.com/job/ramen-pull-requests/1/consoleFull

**Pull request closed without merging**